lua/lspconfig/server_configurations/daedalus_ls.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+local bin_name = 'DaedalusLanguageServer'
```

Comment:
inline

Reply:
I'm not sure what the request is but I guess I should add a newline between right? If yes then why you call it like that?

Reply:
I mean can just in cmd after win32 condition removed.

Reply:
Of course! I didn't see it since you highlighted those 2 lines together :D Fixed.